.github/workflows/vercel_deploy.yml

```diff
@@ -0,0 +1,36 @@
+name: Vercel deploy
+
+on:
+  push:
+    branches:
+      - main
+      - vercel-deploy-2  # for initial test
+  # manual trigger
+  workflow_dispatch:
```

Comment:
Sorry, does this mean this task *is* triggered manually or is that a way *to* trigger it manually?

Reply:
It can optionally be triggered manually in addition to automatically.